docs/src/components/PageHeader.js

```diff
@@ -1,15 +1,14 @@
 // @flow
 import React from 'react';
 import { Box, Text, Heading, Link } from 'gestalt';
-import Markdown from './Markdown';
+import Markdown from './Markdown.js';
 
 type Props = {|
   name: string,
   description?: string,
 |};
 
-const gestaltPath = component =>
-  `/packages/gestalt/src/${component}/${component}.js`;
+const gestaltPath = component => `/packages/gestalt/src/${component}.js`;
```

Comment:
Just noticed this, not a result of your changes, but a "bug" we've had, this is adding 2 `/` characters because of the way we do string templating
`https://github.com/pinterest/gestalt/blob/master//packages/gestalt/src/Checkbox/Checkbox.js`